stable/rocketchat/requirements.yaml

```diff
@@ -1,4 +1,4 @@
 dependencies:
 - name: mongodb
-  version: 4.2.2
+  version: 4.3.x
```

Comment:
See syntax for versionranges: https://github.com/helm/helm/blob/master/docs/chart_best_practices/requirements.md

Reply:
Sure, and looking further down that page: https://github.com/Masterminds/semver#checking-version-constraints :-) .x is a valid wildcard ;-)